src/main/export.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-plusplus */
+import { join } from 'path';
+import { Project, Transcription } from '../sharedTypes';
+import { padZeros, integerDivide, writeFile, mkdir } from './util';
+
+const secondToTimestamp = (num: number) => {
+  return `${padZeros(integerDivide(num, 3600), 2)}:
+          ${padZeros(integerDivide(num, 60), 2)}:
+          ${padZeros(integerDivide(num, 1), 2)}:
+          ${padZeros(integerDivide(num, 0.01), 2)}`;
+};
+
+const constructEDL = (title: string, transcription: Transcription) => {
+  let output = `TITLE: ${title}\nFCM: NON-DROP FRAME\n\n`;
```

Comment:
Can you add a link to your write up (I think you made a write up?) on what the structure of an EDL is like? A G Drive link ideally

Reply:
Not me, I think Max wrote that